bibtexparser/bibdatabase.py

```diff
@@ -64,6 +70,10 @@ def __init__(self):
         #: List of BibTeX preamble (`@preamble{...}`) blocks.
         self.preambles = []
 
+        self.not_update_by_crossref = [
```

Comment:
`not_updated_by_crossref`

This also deserves some comment to explain its purpose.

Reply:
Fixed in 66144482d40b819abab87f774895211035dadd4d.